.github/workflows/autocorrect-lint.yml

```diff
@@ -0,0 +1,36 @@
+name: AutoCorrect Lint
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "files/zh-cn/**"
+      - ".github/workflows/autocorrect-lint.yml"
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get changed files
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BASE_SHA: ${{ github.event.pull_request.base.sha }}
+          HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+        run: |
+          # Use the GitHub API to get the list of changed files
```

Comment:
Maybe something to consider for future, it looks like we're [reusing this logic to check files](https://github.com/mdn/content/blob/221a88897598f0ad73f3e80c4a6211992264dcf5/.github/workflows/pr-check_markdownlint.yml#L22-L29) and it seems to be one of the more efficient ways of doing it after trial and error. We can consider adding it to the [shared workflows](https://github.com/mdn/workflows) repository where you can pass in a path for the `egrep` command. I can imagine us using it like:

```yml
  steps:
    - name: Get changed files
      uses: mdn/workflows/.github/workflows/list-changed-files.yml@main
      with:
        # don't run on forks
        target-repo: "mdn/translated-content"
        # only list these paths
        paths: "^files/zh-cn/"
```

Reply:
Hi @bsmth. I did consider this before. And if I have understood [this](https://docs.github.com/en/actions/using-workflows/reusing-workflows). We could use this as a reusable job only. And in this job, we should set an output for using it in another job.

Or we should create a new action, so we could use this within a single job.

Reply:
> In this job, we should set an output for using it in another job.

I think that's how the old action we were using did it, see https://github.com/tj-actions/changed-files/blob/1a3ea1d3240a68636af870fb623537087c4aaea2/action.yml#L111-L174

Reply:
Yes, but it seems that GitHub only support reusable workflow, so we may have two jobs to deal with it:

```yml
jobs:
  get-changed-files:
    uses: mdn/workflows/.github/workflows/list-changed-files.yml@main
    with:
      target-repo: "mdn/translated-content"
      paths: "^files/zh-cn/"
    secrets:
      token: ${{ secrets.GITHUB_TOKEN }}

  some-job:
    runs-on: ubuntu-latest
    needs: get-changed-files
    if: ${{ needs.get-changed-files.outputs.files }}
    steps:
      # do sth.
```

Reply:
Maybe this is good to record in a GitHub discussion

Reply:
create a discussion as https://github.com/orgs/mdn/discussions/327

Reply:
I'm going to resolve this as it's out of scope of the changes 👍🏻